### PR TITLE
Add KeyPass to Readme

### PR DESCRIPTION
**Pull request by yogeshpaliyal**
# Pull Request Template

## Description

The modern password manager does not store your credentials on servers so your passwords are in your hands, no need to worry.

## Screenshots:

<div style="display: inline-block"  align="center">

</div>

## Additional Notes:

Add any other notes or comments here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ In this repository, you'll find an eclectic collection of projects created by ou
 
 | Project Name | Author | Description | Technologies Used | Repo Link | PlayStore Link |
 |--------------|--------|-------------|-------------------|-----------|----|
-| **Coming Soon** |  |  |  |  | |
+| KeyPass | Yogesh Choudhary Paliyal | Open-source & offline password manager. Store, manage, take control securely. | Compose, Hilt, material 3, MVI (Redux), Room, Jetpack Data Store  | [Github](https://github.com/yogeshpaliyal/KeyPass) | [PlayStore](https://play.google.com/store/apps/details?id=com.yogeshpaliyal.keypass) |
```
Comment (anandwana001) [inline README.md]:
```suggestion
| KeyPass | Yogesh Choudhary Paliyal | Open Source Project & An Offline Password Manager. Store, manage, and take control securely. | Jetpack Compose, Hilt, Material 3, MVI (Redux), Room, Jetpack Data Store  | [Github](https://github.com/yogeshpaliyal/KeyPass) | [PlayStore](https://play.google.com/store/apps/details?id=com.yogeshpaliyal.keypass) |
```

WDYT?

Comment (yogeshpaliyal) [reply README.md]: Looks good

File: README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ In this repository, you'll find an eclectic collection of projects created by ou
 
 | Project Name | Author | Description | Technologies Used | Repo Link | PlayStore Link |
 |--------------|--------|-------------|-------------------|-----------|----|
-| **Coming Soon** |  |  |  |  | |
+| KeyPass | Yogesh Choudhary Paliyal | Open Source Project & An Offline Password Manager. Store, manage, and take control securely. | Jetpack Compose, Hilt, Material 3, MVI (Redux), Room, Jetpack Data Store  | [Github](https://github.com/yogeshpaliyal/KeyPass) | [PlayStore](https://play.google.com/store/apps/details?id=com.yogeshpaliyal.keypass) |
```
Comment (anandwana001) [inline README.md]:
```suggestion
| KeyPass | Yogesh Choudhary Paliyal | Open Source Project & An Offline Password Manager. Store, manage, and take control securely. | Jetpack Compose, Hilt, Material 3, MVI (Redux), Room, Jetpack Data Store  | [GitHub](https://github.com/yogeshpaliyal/KeyPass) | [PlayStore](https://play.google.com/store/apps/details?id=com.yogeshpaliyal.keypass) |
```

**Review by anandwana001 — COMMENTED** [review]
Good to merge,

one more thing

File: README.md (hunk 2 above)
Comment (yogeshpaliyal) [reply README.md]: Done

**Pull request merged**